### PR TITLE
Feature/finp3-734 send raw

### PR DESCRIPTION
**Pull request by fulara**
Changes implemented in fix8 for our raw encode version.

### CODE REVIEW

File: include/fix8/session.hpp [hunk 1]
```diff
@@ -637,6 +645,10 @@ class Session
 	 			with the reamining messages in the vector still allocated */
 	F8API virtual size_t send_batch(const std::vector<Message *>& msgs, bool destroy=true);
 
+    F8API bool send_raw(std::function<std::size_t(const Header&, const char*)> encode, char* buffer);
```
Comment (kepam) [inline include/fix8/session.hpp]: Bad indentation.

File: runtime/session.cpp [hunk 2]
```diff
@@ -941,20 +941,47 @@ int Session::modify_header(MessageBase *msg)
...
-                  _sid.get_targetCompID().get().data(),
-                  sending_time_buffer,
-                  _next_send_seq++};
-	auto size = encode(header, buffer);
-	return _connection->send(buffer, size);
+bool Session::send_raw_process(std::function<std::size_t(const Header&)> encode, const char* buffer)
+{
+    try
+    {
+        char sending_time_buffer[64];
```
Comment (kepam) [inline runtime/session.cpp]: I would probably go with 32, or even 24
As looking here
https://github.com/IG-Group/fix8/blob/master/include/fix8/field.hpp
this date have at most
"YYYYMMDD-HH:MM:SS.mmm"
22 bytes.

Comment (fulara) [reply runtime/session.cpp]: changed to 32, just because i was scared of 24.

File: include/fix8/session.hpp (hunk 1 above)
Comment (fulara) [reply include/fix8/session.hpp]: i changed all 4x spaces in new code to tabs

**Pull request merged**